_studio/mfx_lib/decode/h264/src/mfx_h264_dec_decode.cpp

```diff
@@ -93,9 +93,9 @@ struct ThreadTaskInfo
     mfxFrameSurface1* surface_work = nullptr;
     mfxFrameSurface1* surface_out  = nullptr;
 
-    ThreadTaskInfo(mfxFrameSurface1* work, mfxFrameSurface1* out)
-        : surface_work(work)
-        , surface_out(out)
+    ThreadTaskInfo(mfxFrameSurface1* work, mfxFrameSurface1* out)
+        : surface_work(work)
+        , surface_out(out)
```

Comment:
Sorry. I cannot find what are changed? And it seems not be related with "VDBOX+SFC for 1->1 transcode scenarios". Could we add it into another commit?